### PR TITLE
Darkmode for single file

### PR DESCRIPTION
**Pull request by olibu**
Support for dark mode in single file.
CSS is not modified any more. Themes are based on css conditions instead.
Darkmode is activated earlier now, because pages has flickered on when triggered in index.js

### CODE REVIEW

**Comment by lapo-luchini** [comment]
Thanks! This closes #89.

File: index.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
-<html>
+<html data-theme="dark">
```
Comment (lapo-luchini) [inline index.html:2]: @olibu is this necessary?
(and it is correct when OS theme is light? or would cause flickering on the other way?)

**Comment by lapo-luchini** [comment]
I transplanted your changes [to a new branch](https://github.com/lapo-luchini/asn1js/tree/github-90) (should be 1:1 and in your name).

File: index.html (hunk 1 above)
Comment (lapo-luchini) [reply index.html:2]: Thinking about it, I'd say it's fine as it is.

**Comment by lapo-luchini** [comment]
Merged in f03d4033df69ef7926ac85c59f820e2d8f137bd1.

File: index.html (hunk 1 above)
Comment (olibu) [reply index.html:2]: I verified it in dark and light mode to work without flickering. However, I set dark as the default. When testing I recognized that the dark flash is not as disturbing as the light flash. 
Flickering stopped by separating the function into the theme.js

**Pull request closed without merging**